src/client/components/MapHeader.tsx

```diff
@@ -272,6 +282,7 @@ const MapHeader = ({
                   }}
                   sx={{ width: "150px" }}
                   defaultValue={paintBrushSize}
+                  value={paintBrushSize}
```

Comment:
This line is causing a console error (https://fb.me/react-controlled-components):
![image](https://user-images.githubusercontent.com/4432106/127668728-fb703cbc-9187-413f-8366-9fac609ff42f.png)

```suggestion
                  value={paintBrushSize}
```